pkg/store/sandbox/sandbox.go

```diff
@@ -41,6 +45,49 @@ type Store struct {
 	// TODO(random-liu): Add trunc index.
 }
 
+// NetNS holds network namespace for sandbox
```

Comment:
1) Can we move this to a separate file? netns.go or something?
2) Why following functions are member function of `Sandbox` instead of `NetNS`?

Basically, I want Sandbox/Container to be a simple collection of resources, each resource is thread safe, and you should be able to operate them separately without worrying about any concurrent issue. However, `CreateNetNS` doesn't seem so. It seems better to:
  a) Have a standalone NewNetNS() function;
  b) Add `Remove` and `GetPath` function to `NetNS`. Actually, we don't even need the `GetPath` function, because we could always get it from sandbox metadata.
  c) In the future we'll add `LoadNetNS()` to load existing `NetNS`, which will be useful for restart recovery.